### DISCUSSION TITLE
Not authenticated Error

### DISCUSSION THREAD

**Discussion by siddiqkaithodu** [gavvvr/obsidian-imgur-plugin#92] [Getting help]
Is this issue Fixed? I am having this
![image](https://github.com/user-attachments/assets/7655c0a3-73a9-4da1-9a3f-f5c019e8ec6e)

_Originally posted by @siddiqkaithodu in https://github.com/gavvvr/obsidian-imgur-plugin/issues/80#issuecomment-2326325969_
            
@gavvvr

**Comment by gavvvr** [comment]
Hi @siddiqkaithodu 
Yes, the issue number 80 was fixed. Did you check what's in the console?
What happens if you press "Authenticate" button on the right?
